src/message/index.js

```diff
@@ -7,4 +7,5 @@ const topicDescriptorProto = protons(require('./topic-descriptor.proto.js'))
 
 exports = module.exports
 exports.rpc = rpcProto
+exports.RPC = rpcProto.RPC
 exports.td = topicDescriptorProto
```

Comment:
Should we also just export the message content here?

Reply:
I just realised that we do not use this here. This should only be used by the base class `libp2p-pubsub`. Can we also get this out?

Reply:
:+1:  will remove